internal/decryption/encrypted_audit_log.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package decryption
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/mongodb/mongocli/internal/decryption/keyproviders"
+)
+
+type AuditRecordType string
+type AuditLogFormat string
+
+type AuditLogLineKeyStoreIdentifier struct {
```

Comment:
[nit] since you are using the cli printer you should treat this struct as potential json and add tags so users doing `mongocli ops-manager listKeyProvider --file log.gz -o json` get a consistent output

Reply:
I added the json annotations, thanks